queries.sql

```diff
@@ -155,4 +155,12 @@ FULL JOIN animals
 ON animals.id = visits.animals_id
 FULL JOIN vets
 ON vets.id = visits.vet_id
-WHERE vets.name = 'Maisy Smith';
\ No newline at end of file
+WHERE vets.name = 'Maisy Smith';
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
```

Comment:
Kindly, add screenshot for this `SELECT COUNT(*) FROM visits where animal_id = 4;` after the improvement was introduced. I can't find it on the PR description while the other two where successfully implemented and their screenshot was added.

**Note:** Partner with the lowest commit history has to fix this issue to keep the commit history even.

Reply:
@Mre55 
We modified the SELECT statement to give us an improved execution time. If you check the picture properly, we added it but we only included the animal_id and date_of_visit in the query because that was all the information that we require since the vet_id is the same and the id is always increasing. The goal of the project was to find a way to reduce the execution time below 1second.
![Screenshot (109)](https://user-images.githubusercontent.com/98261268/178291547-a33453a4-3b1e-46a4-b594-e313e955c8a9.png)